packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -453,38 +443,11 @@ export class SummarizerNode implements IRootSummarizerNode {
 					return { latestSummaryUpdated: false };
 				}
 
-				// Fetch the latest snapshot and refresh state from it. Note that we need to use the reference sequence number
-				// of the fetched snapshot and not the "summaryRefSeq" that was passed in.
-				const { snapshotTree, snapshotRefSeq: fetchedSnapshotRefSeq } =
-					await fetchLatestSnapshot();
-
-				// Possible re-entrancy. We may have updated latest summary state while fetching the snapshot. If the fetched
-				// snapshot is older than the latest tracked summary, ignore it.
-				if (this.referenceSequenceNumber >= fetchedSnapshotRefSeq) {
-					eventProps.latestSummaryUpdated = false;
-					event.end(eventProps);
-					return { latestSummaryUpdated: false };
-				}
-
-				await this.refreshLatestSummaryFromSnapshot(
-					fetchedSnapshotRefSeq,
-					snapshotTree,
-					undefined,
-					EscapedPath.create(""),
-					correlatedSummaryLogger,
-					readAndParseBlob,
-				);
-
+				// Note that we did not track this summary, but that the latest summary was updated.
```

Comment:
I don't see where we are updating latest summary.  If it's true that we're not, then let's update this and the doc comment above accordingly.  Makes sense not to since we're about to close.

Reply:
And maybe even rename this function to something like `findPendingSummaryMatchingSummaryAck`